src/porepy/models/material_constants.py

```diff
@@ -567,16 +567,18 @@ def fracture_tangential_stiffness(self) -> number:
             self.constants["fracture_tangential_stiffness"], "Pa*m^-1"
         )
 
-    def maximum_fracture_closure(self) -> number:
-        """The maximum closure of a fracture [m].
+    def maximum_elastic_fracture_opening(self) -> number:
```

Comment:
Since this is API breaking, should we keep the method `maximum_fracture_closure` but let it return an error message?